### PR TITLE
Add testcase for context switcher

### PR DESCRIPTION
**Pull request by fbalak**
Related design: https://redhat.invisionapp.com/share/8QCOEVEY9#/screens/244820318

### CODE REVIEW

File: web/tendrl/cluster_context_switcher.rst [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+       Check that all volumes from cluster that is currently selected are present in list of volumes.
+   :result:
+       All volumes are listed and no volume is missing or is extra.
+
+.. test_action::
+   :step:
+       | Click on ``Context switcher`` menu and select different cluster.
+       | Repeat with all clusters.
+   :result:
+       Current page should be rendered again with volumes for currently selected cluster.
```
Comment (ltrilety) [inline web/tendrl/cluster_context_switcher.rst]: Not current page for sure, if I choose a different cluster it goes to the `Hosts` of the cluster and I have to switch it to `Volumes` to get list of cluster volumes.

File: web/tendrl/cluster_context_switcher.rst [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+   :step:
+       Check that correct cluster is still selected.
+   :result:
+       Correct ``cluster id`` is shown in context switcher.
+
+.. test_action::
+   :step:
+       Click on ``Context switcher`` menu and select different cluster.
+   :result:
+       Current page should be redirected to ``Volumes`` page for given cluster.
```
Comment (ltrilety) [inline web/tendrl/cluster_context_switcher.rst]: Actually `Hosts` is the default one.

File: web/tendrl/cluster_context_switcher.rst [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+       Check that all tasks from cluster that is currently selected are present in list of tasks.
+   :result:
+       All tasks are listed and no task is missing or is extra.
+
+.. test_action::
+   :step:
+       | Click on ``Context switcher`` menu and select different cluster.
+       | Repeat with all clusters.
+   :result:
+       Current page should be rendered again with tasks for currently selected cluster.
```
Comment (ltrilety) [inline web/tendrl/cluster_context_switcher.rst]: The same as for `Volumes` the default page for any cluster is `Hosts` page.

File: web/tendrl/cluster_context_switcher.rst [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+       Check that all events from cluster that is currently selected are present in list of events.
+   :result:
+       All events are listed and no event is missing or is extra.
+
+.. test_action::
+   :step:
+       | Click on ``Context switcher`` menu and select different cluster.
+       | Repeat with all clusters.
+   :result:
+       Current page should be rendered again with events for currently selected cluster.
```
Comment (ltrilety) [inline web/tendrl/cluster_context_switcher.rst]: The same as for `Volumes` the default page for any cluster is `Hosts` page.

File: web/tendrl/cluster_context_switcher.rst [hunk 5]
```diff
@@ -0,0 +1,140 @@
...
+   :result:
+       All tasks are listed and no task is missing or is extra.
+
+.. test_action::
+   :step:
+       | Click on ``Context switcher`` menu and select different cluster.
+       | Repeat with all clusters.
+   :result:
+       Current page should be rendered again with tasks for currently selected cluster.
+
```
Comment (ltrilety) [inline web/tendrl/cluster_context_switcher.rst]: You are checking `brick details` for host and volume, what about checking the related `Task Details` for a task.

**Review by ltrilety — CHANGES_REQUESTED** [review]

**Review by ltrilety — APPROVED** [review]
LGTM

File: web/tendrl/cluster_context_switcher.rst [hunk 6]
```diff
@@ -0,0 +1,134 @@
...
+:authors: 
+          - fbalak@redhat.com
+
+Description
+===========
+
+Change context of tendrl UI - show items in UI related to selected cluster.
+Context switcher is the dropdown menu rendered in masthead for each page
+except ``Clusters`` in UI.
+
```
Comment (mbukatov) [inline web/tendrl/cluster_context_switcher.rst]: Could you link the design https://redhat.invisionapp.com/share/8QCOEVEY9#/screens/244820318 here? Also note that you can add the link url into https://github.com/usmqe/usmqe-testdoc/blob/master/references.rst file and just use it from there (as the references file is included in every rst file in the repo).

File: web/tendrl/cluster_context_switcher.rst [hunk 7]
```diff
@@ -0,0 +1,134 @@
...
+   :result:
+       UI behaves as is expected and shows items related to selected context.
+
+.. test_action::
+   :step:
+       | Click on ``Context switcher`` menu and select different cluster.
+       | Repeat with all clusters and from all locations in UI:
+       | ``Hosts``, ``Volumes``, ``Tasks`` and ``Events``.
+   :result:
+       Current page is redirected to ``Hosts`` page for given cluster.
```
Comment (mbukatov) [inline web/tendrl/cluster_context_switcher.rst:132]: How is this different from the previous test action?

Comment (fbalak) [reply web/tendrl/cluster_context_switcher.rst:132]: Starting point of previous step is `All clusters` page and all previous steps should be carried with cluster selected on this page.
In this step user tests drop down menu in the masthead and page redirection for its options.

File: web/tendrl/cluster_context_switcher.rst [hunk 8]
```diff
@@ -0,0 +1,134 @@
...
+.. test_action::
+   :step:
+       Check that there is no left panel shown after imports.
+   :result:
+       There is no left panel with menu visible on left side and context
+       switcher is set to ``All clusters``.
+
+.. test_action::
+   :step:
+       Click on the name of first imported cluster in the cluster list.
```
Comment (mbukatov) [inline web/tendrl/cluster_context_switcher.rst]: Ah, so could you highlight that by "cluster list" you don't mean list of clusters in context switcher?

**Review by mbukatov — APPROVED** [review]

**Pull request merged**